jenkins_job_manager/cli.py

```diff
@@ -148,14 +150,25 @@ def output_format(line):
 
 @jjm.command(name="plan")
 @click.option("--skip-pager", is_flag=True)
+@click.option("--json", help="Display the plan report in json format.")
+@click.option("--yaml", help="Display the plan report in yaml format.")
```

Comment:
probably not the ideal interface. Other CLI apps use `--output=json` or `--output=yaml` for this sort of thing.

Additionally, if there's valid and invalid inputs to these, we'd want them caught before going any further. 
Click's "choice" options are good for this: https://click.palletsprojects.com/en/7.x/options/#choice-options